src/content/docs/tutorial-customer-experience/intro-to-customer-experience.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Improve your customers' digital experience with New Relic" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/qf-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/quality-foundation-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide
+---
+
+import cxWhatYouCanMeasureNr from 'images/cx_diagram_what-you-can-measure-nr.webp'
+
+Digital customer experience refers to the quality of how your users interact with your service across all your digital touch points (web, mobile, and IoT). Because it has so many moving parts within the concept, it can be difficult to digest when taken as whole. You need to manage the availability of your service, its performance to ensure that it's usable, the quality of the content that gives users the information they need, and the relevance of your products and content so that users remain engaged.
+
+New Relic can help you make managing this much more approachable by measuring and improving performance in a way that better aligns to the end user experience. This guide covers the procedures that help you understand your digital customer experience in a meaningful way:
+
+* Look at customer experience in relation to:
+  * Global functions, such as search and login
+  * Lines of business
+  * Regions
+* Report back to business stakeholders on what they care about
+* Prioritize what to improve next
+
+<img
+  title="Digital Customer Experience (DCX): What you can measure with New Relic"
+  alt="Digital customer experience"
+  src={cxWhatYouCanMeasureNr}
+/>
+
+## Prerequisites
```

Comment:
How about "Check your requirements"?

Reply:
Also, looks like you're missing an h2 ID here

Reply:
Good callouts, I've modified this. Thank you!